components/style/properties/longhand/ui.mako.rs

```diff
@@ -53,6 +53,16 @@ ${helpers.predefined_type("-moz-window-transform", "Transform",
                           internal=True,
                           spec="None (Nonstandard internal property)")}
 
+${helpers.predefined_type("-moz-window-transform-origin",
+                          "TransformOrigin",
```

Comment:
nit: Alignment is off by one too (or so it seems)

Reply:
Looks like moz-window-opacity also had this problem, feel free to fix that up too.

Reply:
Err, actually alignments are not off, and it's github's font or something it seems? Disregard those.